src/main/java/app/model/agents/DirectionFollowAgent/GoToTargetAgent.java

```diff
@@ -0,0 +1,94 @@
+package app.model.agents.DirectionFollowAgent;
+
+import app.controller.graphicsEngine.Ray;
+import app.controller.linAlg.Angle;
+import app.controller.linAlg.Vector;
+import app.model.Move;
+import app.model.Type;
+import app.model.agents.Agent;
+import app.model.agents.AgentImp;
+import app.model.agents.World;
+
+public class GoToTargetAgent extends AgentImp
```

Comment:
Rename to `TargetAgent`, "GoTo" is redundant

Reply:
will do